packages/common/src/utils/internal/getDefaultEntitySettings.ts

```diff
@@ -0,0 +1,12 @@
+import { EntitySettingType, IEntitySettings } from "../../discussions";
+
+export function getDefaultEntitySettings(
```

Comment:
We should make this a bit more intelligent to support expanding the settings out to other entity types, and composing different default values together. Lets change `type: EntitySettingType` to `entityType: HubEntityType` and have it populate the correct `EntitySettingType` for the given `entityType`. Something like the following should work:

```jsx
import { EntitySettingType, IEntitySetting, IEntitySettings } from "../../discussions";
import { HubEntityType } from '../../core/types';

const DISCUSSION_SETTINGS: IEntitySettings = {
  discussions: {
    allowedChannelIds: null,
    allowedLocations: null,
  },
};

const SETTINGS_TYPE_BY_ENTITY_TYPE: Record<HubEntityType, EntitySettingType> = {
  discussion: EntitySettingType.CONTENT,
  site: null,
  project: null,
  initiative: null,
  initiativeTemplate: null,
  page: null,
  content: null,
  org: null,
  group: null,
  template: null,
};

const DEFAULT_ENTITY_SETTINGS_BY_ENTITY_TYPE: Record<HubEntityType, IEntitySettings> = {
  discussion: {
    ...DISCUSSION_SETTINGS,
  },
  site: null,
  project: null,
  initiative: null,
  initiativeTemplate: null,
  page: null,
  content: null,
  org: null,
  group: null,
  template: null,
};

export function getDefaultEntitySettings(
  entityType: HubEntityType
): Partial<IEntitySetting> {
  if (!SETTINGS_TYPE_BY_ENTITY_TYPE[entityType]) {
    throw new Error(`no entity settings type defined for ${entityType}`);
  }
  if (!DEFAULT_ENTITY_SETTINGS_BY_ENTITY_TYPE[entityType]) {
    throw new Error(`no default entity settings defined for ${entityType}`);
  }
  return {
    type: SETTINGS_TYPE_BY_ENTITY_TYPE[entityType],
    settings: { ...DEFAULT_ENTITY_SETTINGS_BY_ENTITY_TYPE[entityType] }
  };
}
```